projects/go-style-guide/src/app/shared/components/sub-nav/sub-nav.component.html

```diff
@@ -1,5 +1,6 @@
 <header class="go-page-title">
   <h1 class="go-heading-1">{{ subNavService.pageTitle }}</h1>
+  <go-button (handleClick)="linkToSource()" buttonVariant="neutral"> Link To Source </go-button>
```

Comment:
We could do the same thing here with adding an `<a>` in the html to avoid the typescript file change.

```
<a
  [href]="subNavService.linkToSource"
  target="_blank">
  <go-button buttonVariant="neutral">View on GitHub</go-button>
</a>
```